js/multimodaltextbox/shared/MultimodalTextbox.svelte

```diff
@@ -229,7 +229,32 @@
 		event.preventDefault();
 		dragging = false;
 		if (event.dataTransfer && event.dataTransfer.files) {
-			upload_component.load_files(Array.from(event.dataTransfer.files));
+			const files = Array.from(event.dataTransfer.files);
+
+			if (file_types) {
+				const valid_files = files.filter((file) => {
```

Comment:
I think this validation logic is repeated in UploadButton/Upload components. Would be good to standardize in a common location at this point.